.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
```

Comment:
We can make this a bit longer to start. 90 Days?

Reply:
done